server/trillian_log_server/main.go

```diff
@@ -18,27 +18,28 @@ package main
 
 import (
 	"flag"
-	_ "net/http/pprof"
-
-	_ "github.com/go-sql-driver/mysql" // Load MySQL driver
 
 	"github.com/golang/glog"
 	"github.com/google/trillian"
 	"github.com/google/trillian/cmd"
 	"github.com/google/trillian/crypto/keys"
 	"github.com/google/trillian/extension"
-	_ "github.com/google/trillian/merkle/objhasher" // Load hashers
-	_ "github.com/google/trillian/merkle/rfc6962"   // Load hashers
 	"github.com/google/trillian/monitoring"
 	"github.com/google/trillian/monitoring/prometheus"
-	"github.com/google/trillian/quota"
-	mysqlq "github.com/google/trillian/quota/mysql"
 	"github.com/google/trillian/server"
 	"github.com/google/trillian/server/interceptor"
 	"github.com/google/trillian/storage/mysql"
 	"github.com/google/trillian/util"
 	"golang.org/x/net/context"
 	"google.golang.org/grpc"
+
+	_ "net/http/pprof"
```

Comment:
Done.

Personally I dislike the comments, as the packages should clearly document what the import side effects are (pprof does this fairly well). Plus the "_" is a strong sign in itself.